test/config.yaml

```diff
@@ -1,4 +0,0 @@
-sampletable: pasilla_sampletable.tsv
-sample_dir: pasilla
-assembly: dm6
-data_dir: /data/LCDB/references
```

Comment:
I guess leave it for now. I'm planning on building a more general way of handling test datasets, so I'll shuffle stuff around during that.